config/schedule.rb

```diff
@@ -173,6 +174,12 @@
     at: '11:00am',
     interruptable: true,
   },
+  {
+    task: 'driver:hmis:process_activity_logs',
+    frequency: 60.minutes,
+    trigger: hmis_trigger,
+    interruptable: true,
+  },
```

Comment:
Instead of adding a new task, can this be added to `task: 'grda_warehouse:hourly'`?  Each of these tasks spins up a new container which requires additional capacity and potentially adds pressure to the existing DB that is outside of what's expected.  If it doesn't need to run exactly each hour, you can add it to the hourly task and have it queue up a delayed job that will be processed when the worker gets there.  Sometimes those also have to spin up an extra container, but they are limited in the number of containers they'll spin up, so the DB use should be somewhat limited.